### ISSUE TITLE
Drag down: can't return back to the original cell

### ISSUE THREAD

**Issue by andrewQwer** [handsontable/handsontable#4233]
Hi, 

When using [Drag down](https://docs.handsontable.com/pro/1.10.2/demo-drag-down.html#page-vertical-without-creating-rows) I can't return selection to the original cell when using `direction: 'vertical'`. For horizontal direction and default options it works.
Demo: http://jsfiddle.net/andrewQwer/o4rwf84j/3/
Video example: https://www.screencast.com/t/c93kbsfBkiI

**Comment by AMBudnik** [comment]
Hi @andrewQwer 

`direction: 'horizontal'` is not a valid combination for `fillHandle`, you can pick from `false`, `true` and `vertical`. However, you are right when it comes to the `vertical` dragging - it doesn't allow you to get back to original cell (prevent replicating a value).

Thanks for sharing.

**Comment by andrewQwer** [comment]
Hi @AMBudnik ,

You say that 'horizontal' is not valid, but documenation says it's valid: 
https://docs.handsontable.com/pro/1.10.2/Options.html#fillHandle
```
fillHandle: { // enable plugin in vertical direction and with autoInsertRow as false
  autoInsertRow: false,
  direction: 'vertical' // 'vertical' or 'horizontal'
}
```

**Comment by AMBudnik** [comment]
You're - my bad!

**Comment by AMBudnik** [comment]
The issue is still replicable in v `2.0.0.` I also couldn't find a version which allowed us to keep focus only on the original cell.

**Comment by AMBudnik** [comment]
It remains the same in `7.4.2` http://jsfiddle.net/t9gscxov/ , when vertical direction is allowed we cannot return with the fill handle to the original cell.

**Comment by AMBudnik** [comment]
I'm glad to announce that this issue has been fixed with the rev11 of 8.0.0-beta2 http://jsfiddle.net/AMBudnik/sepuoyq0/
Thank you for sharing @andrewQwer I hope that you still can benefit from this fix.

**Comment by andrewQwer** [comment]
Nice, thank you :)

**Comment by AMBudnik** [comment]
Hi @andrewQwer 

Hi [Name]/ @mention,

I have good news! We've just released 8.0.0-beta2 that fixes the mentioned issue. We won't be closing this ticket now. It will be closed after we release the final 8.0.0 version of Handsontable.

Here’s a full list of changes https://github.com/handsontable/handsontable/releases/tag/8.0.0-beta.2
NPM https://www.npmjs.com/package/handsontable/v/8.0.0-beta.2 

All feedback's appreciated. Thank you for your input.

**Comment by AMBudnik** [comment]
Hi @andrewQwer 

we've just released the stable v8.0.0.

CDN https://cdn.jsdelivr.net/gh/handsontable/handsontable@latest/dist/handsontable.full.css
https://cdn.jsdelivr.net/gh/handsontable/handsontable@latest/dist/handsontable.full.js
NPM npm i handsontable
Release Notes https://github.com/handsontable/handsontable/releases  

I highly encourage you to test the new version and leave some feedback. I hope that the amount of work that we put on the v8 will help you to improve your project.
